install/kubernetes/helm/README.md

```diff
@@ -1,9 +1,3 @@
 # Installation using Helm
 
 Please follow the installation instructions from [istio.io](https://istio.io/docs/setup/kubernetes/install/helm/).
-
```

Comment:
How about: 'new or risky features should go to istio/installer - only critical bugfixes in here'.